package.json

```diff
@@ -17,12 +17,6 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
-  },
```

Comment:
- [ ] There is an error on the terminal which may be related to this deleted eslintConfig here, kindly bring the eslintConfig back and fix the error 👍. Your app displays on the browser, however, we need to make the error on the terminal go away so that your work doesn't have any errors. See the error message below;

<img width="390" alt="chrome_HZNsXuMj0b" src="https://user-images.githubusercontent.com/81468675/164061499-cbb8739e-220c-440d-9fd7-4fcf97476da8.png">

Reply:
Thank you for the positive review @julie-ify . However I did fix this error when I submitted the pull request. If you don't mind, could you please kindly check it again as it is running perfectly on my end. If not, then I will revert to the old settings. Thank you!

Reply:
![log](https://user-images.githubusercontent.com/97033479/164065455-6b7efde2-01d4-4ef4-83c9-d30969ed205e.PNG)

Reply:
If you've fix it then you can request another review. I'm not available now to check it okay.